modules/core/shared/src/main/scala/cron4s/expr/nodes.scala

```diff
@@ -47,7 +47,7 @@ sealed trait Node[F <: CronField] {
 final class EachNode[F <: CronField] private (val unit: CronUnit[F]) extends Node[F] {
   override def equals(other: Any): Boolean =
     other match {
-      case _: EachNode[F] => true
+      case _: EachNode[_] => true
```

Comment:
address compiler warning like `type is unchecked since it is eliminated by erasure`